django_module/tests/functional/test_order.py

```diff
@@ -92,8 +100,12 @@ def test_order_process_fail_if_payment_is_not_confirmed(db, data):
 
 
 def test_order_process_fail_if_location_is_not_available(db, data):
-    product, store, store_item, customer, order, order_item, payment = data
-    order.location = 'Astana'
+    product, city, location, store, store_item, customer, order, order_item, payment = data
+    order = Order.objects.create(
+        price=10,
+        is_paid=True,
+        city='Astana'

```

Comment:
city=City.objects.create(name='Astana')

Reply:
Test is working
![image](https://user-images.githubusercontent.com/45433350/56338049-285d4500-61ca-11e9-921e-3ef789dfbd89.png)